client/resdef.go

```diff
@@ -55,14 +55,7 @@ type resourceDefinitions struct {
 }
 
 func newResourceDefinitions(c restclient.Config) (*resourceDefinitions, error) {
-	c.APIPath = "/apis"
-	c.ContentConfig = restclient.ContentConfig{
-		GroupVersion: &unversioned.GroupVersion{
-			Group:   "appcontroller.k8s2",
-			Version: "v1alpha1",
-		},
-		NegotiatedSerializer: api.Codecs,
-	}
+	thirdPartyResourceConfigInit(&c)
```

Comment:
this and below duplicates the same in dependencies.go. Could it be refactored somehow?